bfcfg

```diff
@@ -131,7 +131,7 @@ sys_cfg_one_byte()
 #   SYS_ENABLE_SPCR      31            1
 #   SYS_DISABLE_PCIE     32            1
 #   SYS_ENABLE_OPTEE     33            1
```

Comment:
Could you fix these three as well (adding 1)?

Reply:
done